easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.28-GCC-14.2.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'OpenBLAS'
+version = '0.3.28'
```

Comment:
@branfosj We should bump this to `0.3.29` (released 12 Jan 2025)

Reply:
ced7dfd